api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -198,6 +198,7 @@ private Patient createBasicPatient() {
 		// patient.removeAddress(pAddress);
 		
 		patient.setDeathDate(new Date());
+		patient.setDead(true);
 		// patient.setCauseOfDeath("air");
 		patient.setBirthdate(new Date());
 		patient.setBirthdateEstimated(true);
```

Comment:
yes i completely understand, but without  ( patient.setDead(true);) this particular test shows validation exception as you see above that the deathdate has been set to current date (new Date())
which means we are setting the deathdate but not setting the person as dead which is a voilation of the issue in the ticket, isn't it?